CHANGES.rst

```diff
@@ -22,6 +22,11 @@ Olivier Grisel
     thread is not the Python default. Thanks to Roman Yurchak for the
     suggestion.
 
+Karan Desai
+
+    py.test is used to run the tests instead of nosetests.
+    ``python setup.py test`` or ``python setup.py nosetests`` do not work
```

Comment:
I lightly edited this. In particular: in rst you need to use double-backquotes for the equivalent of single backquote in markdown.